File: Examples/VisualBasic/Outlook/NewPSTAddSubfolders.vb
Imports System.IO
Imports Aspose.Email.Mail
Imports Aspose.Email.Outlook
Imports Aspose.Email.Pop3
Imports Aspose.Email
Imports Aspose.Email.Mime
Imports Aspose.Email.Imap
Imports System.Configuration
Imports System.Data
Imports Aspose.Email.Mail.Bounce
Imports Aspose.Email.Exchange
Imports Aspose.Email.Outlook.Pst

Namespace Aspose.Email.Examples.VisualBasic.Email.Outlook

    Public Class NewPSTAddSubfolders
        Public Shared Sub Run()
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_Outlook()
            Dim dst As String = dataDir & Convert.ToString("PersonalStorage.pst")

            If File.Exists(dst) Then
                File.Delete(dst)
            End If

            ' Create new PST
            Dim pst As PersonalStorage = PersonalStorage.Create(dst, FileFormatVersion.Unicode)

            ' Add new folder "Inbox"
            pst.RootFolder.AddSubFolder("Inbox")

            Console.WriteLine(Environment.NewLine + "PST saved successfully at " & dst)
        End Sub
    End Class
End Namespace

File: Examples/CSharp/Outlook/NewPSTAddSubfolders.cs
using System.IO;
using System;
using Aspose.Email;
using Aspose.Email.Storage.Pst;

namespace Aspose.Email.Examples.CSharp.Email.Outlook
{
    class NewPSTAddSubfolders
    {
        public static void Run()
        {
            // The path to the File directory.
            string dataDir = RunExamples.GetDataDir_Outlook();
            string dst = dataDir + "PersonalStorage.pst";

            if (File.Exists(dst))
                File.Delete(dst);
            
            // Create new PST
            PersonalStorage pst = PersonalStorage.Create(dst, FileFormatVersion.Unicode);

            // Add new folder "Inbox"
            pst.RootFolder.AddSubFolder("Inbox");

            Console.WriteLine(Environment.NewLine + "PST saved successfully at " + dst);
        }
    }
}
